onto_spread_ed/release/GenerateHierarchicalSpreadsheetReleaseStep.py

```
@@ -0,0 +1,180 @@
+            hierarchies, ontology = self.build_hierarchy(file)
+
+            wb = openpyxl.Workbook()
+            sheet: Worksheet = wb.active
+
+            height = max(h.height() for h in hierarchies)
+            annotations = list({k for h in hierarchies for k in h.annotations.keys()})
+
+            sheet.append(["ID", "Label"] + [""] * (height - 1) + ["Definition"] + annotations)
+
+            def write_line(n: Node, depth: int) -> None:
+                sheet.append([ontology.get_id_for_iri(n.item)] +
```

Ruff

Function definition does not bind loop variable `sheet`

(B023)